p2p_wallet/Coordinators/TabBar/HomeCoordinator.swift

```diff
@@ -126,8 +126,6 @@ final class HomeCoordinator: Coordinator<Void> {
             return coordinate(
                 to: SendCoordinator(
                     rootViewController: navigationController,
-                    preChosenWallet: nil,
-                    hideTabBar: true,
```

Comment:
`hideTabBar` was default `false`, are you sure deleting this line?

Reply:
yes. I changed the logic of its appearance with this line
'vc.hidesBottomBarWhenPushed = rootViewController.tabBarController != nil'. now we don't need to pass it but it will behave itself

Reply:
previously it was set manually